site/content/en/blog/releases/1.23.0-rc.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "1.23.0 - Release Candidate"
+linkTitle: "1.23.0-rc"
+date: "2022-05-03"
+---
+This is the 1.23.0-rc release of Agones.
+
+This release brings us support for Kubernetes 1.22, so if you are installing this release,  Please remember to upgrade your clusters.
```

Comment:
Can you also please highlight that this is the first release that supports arm architecture. Something like we had in https://agones.dev/site/blog/2021/04/27/1.14.0-initial-windows-support-rollingupdateonready-to-beta/:

"This release introduces initial support for running game servers on linux machines using the ARM architecture! If this is of interest to you, please test it out and give us feedback so that we can ensure that it works reliably before promoting the support to stable."